src/components/List.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import {Crypto} from './Interfaces';
+
+export const List = ({cryptos} /* :Crypto[] */) => {
+  let bull: boolean;
```

Comment:
[critical] rebuild all this bull validation, use useState, also is a very bad practice update variables in the return like you do